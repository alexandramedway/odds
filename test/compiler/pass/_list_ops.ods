do ints = [1, 2, 3, 4]

/* list head tests */
do print(head([]))
do x = head(ints)
do print(x)
do print(x + 1)
