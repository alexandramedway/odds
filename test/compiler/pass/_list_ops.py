ints_0 = [1, 2, 3, 4]
print(([][0] if [] else None))
x_1 = (ints_0[0] if ints_0 else None)
print(x_1)
print((x_1 + 1))
