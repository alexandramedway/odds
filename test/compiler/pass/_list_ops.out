None
1
2
